static/script/devices/mediaplayer/cehtml.js

```diff
@@ -429,6 +430,7 @@ require.def(
                 if (clampedTime !== seconds) {
                     RuntimeContext.getDevice().getLogger().debug("playFrom " + seconds + " clamped to " + clampedTime + " - seekable range is { start: " + this._range.start + ", end: " + this._range.end + " }");
                 }
+                this._sentinelSeekTime = clampedTime;
```

Comment:
Added 'testSeekSentinelClampsTargetSeekTimeWhenRequired' to cehtmlcommontests to cover this. Similar to how it has been done for html5